style.css

```diff
@@ -0,0 +1,67 @@
+body {
+  background-image: url("image/Header bg@2x.jpg");
+}
+
+h1 h3 {
+  color: #0a1529;
+}
+
+h2 {
+  color: #6c7af8;
+}
+
+p {
+  color: #293c5e;
+}
+
+#headline {
+  width: 70%;
+  margin: 0 auto;
+  margin-top: 10%;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.navbar {
+  display: flex;
+  justify-content: space-between;
+  width: 80%;
+  background: #fff;
+  margin: 0 auto;
+  padding: 30px;
+}
+
+.d-navbar {
+  display: flex;
+  list-style-type: none;
+  cursor: pointer;
+}
+
+.a-navbar {
+  text-decoration: none;
+  height: 2em;
+  padding: 0.5em 1em;
+}
+
+.welcome {
+  width: 40%;
+  margin: 0 auto;
+  margin-top: 10%;
+}
+
+.introduction {
+  width: 40%;
```

Comment:
- [ ] `The width: 40%` is making your introduction narrow while we have a lot more space. Please follow the Figma design and have more width for the introduction with some margin on both left and right